feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.ts

```diff
@@ -15,17 +19,31 @@ export class ConfiguratorCartEntryBundleInfoService {
    * @returns {LineItem[]} - Line item array
    */
   retrieveLineItems(entry: OrderEntry): LineItem[] {
-    const lineItems: LineItem[] = [];
+    let lineItems: LineItem[] = [];
     if (entry?.configurationInfos) {
-      entry.configurationInfos
-        .filter(
-          (configurationInfo) =>
-            configurationInfo.configurationLabel ||
-            configurationInfo.configurationValue
-        )
-        .forEach((configurationInfo) =>
+      const configurationInfos: ConfigurationInfo[] = entry.configurationInfos.filter(
+        (configurationInfo) =>
+          configurationInfo &&
+          (configurationInfo.configurationLabel ||
+            configurationInfo.configurationValue)
+      );
+      const firstLabel: string =
+        configurationInfos[0]?.configurationLabel ?? '';
+      const firstValue: string =
+        configurationInfos[0]?.configurationValue ?? '';
```

Comment:
```suggestion
      const firstLabel  =   configurationInfos[0]?.configurationLabel;
      const firstValue  =   configurationInfos[0]?.configurationValue;
```
firstLabel/firstValue can be undefined, no need to initialize them with '' here